src/libraries/System.Net.Mail/src/System.Net.Mail.csproj

```diff
@@ -238,7 +238,7 @@
     <Reference Include="System.Security.Principal.Windows" />
     <Reference Include="System.Threading" />
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true'">
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
     <Reference Include="System.Memory" />
   </ItemGroup>
 </Project>
```

Comment:
@Anipik I added a PNSE implementation for this recently in https://github.com/dotnet/runtime/commit/80fe5bba774dabc47a7334d97f4a7aef2ca274c5 but I didn't know about the DesignTimeBuild requirements so I added it to the first PropertyGroup.

Can you please fix that in this PR?